Tests/Unit/Commenting/FunctionCommentSniffTest.phptest

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- *   This file is part of the ONGR package.
+ * This file is part of the ONGR package.
```

Comment:
Maybe this was a part of failure testing and spaces were inserted for testing reasons?